classes/core/PKPContainer.inc.php

```diff
@@ -333,6 +350,32 @@ protected function settingProxyForStreamContext(): void
         stream_context_set_default($opts);
         libxml_set_streams_context($context);
     }
+
+    /**
+     * Determine if the application is running unit tests.
+     */
+    public function runningUnitTests(): bool
+    {
+        return isset($_SERVER['PKP_MOCK_ENV']);
+    }
+
+    /**
+     * Set the storage directory.
+     */
+    public function useStoragePath(string $path): static
+    {
+        $this->storagePath = $path;
+        $this->instance('path.storage', $path);
+        return $this;
+    }
+
+    /**
+     * Get the path to the storage directory.
+     */
+    public function storagePath($path = ''): string
+    {
+        return ($this->storagePath ?: (new PrivateFileManager())->getBasePath()) . ($path != '' ? "/$path" : '');
+    }
```

Comment:
Some of the Laravel's facades assumes that the container is implementing the `Illuminate\Contracts\Foundation\Application`, which has a lot of extra methods, as we're not doing it yet, I've just added these methods to satisfy the `Log` facade.